extensions/guacamole-auth-cas/src/main/resources/translations/es.json

```diff
@@ -0,0 +1,12 @@
+{
+
+    "DATA_SOURCE_CAS" : {
+        "NAME" : "CAS SSO Backend"
```

Comment:
Is this correct? The Spanish translation for "CAS SSO Backend" is "CAS SSO Backend"? If yes, that's fine, but I wouldn't have expected the Spanish versions of the names of each of these backends to be identical to English.